src/utils/api.js

```diff
@@ -1,13 +1,18 @@
 /* eslint-disable no-unused-expressions */
 import { useUserDataStore } from '../stores/userData.js'
+import { useSessionsStore } from '../stores/sessions.js'
 
 async function fetchApiCall ({ url, method, body = null, header, successCallback = null, failCallback = null }) {
   const store = useUserDataStore()
+  const sessionsStore = useSessionsStore()
 
   const defaultHeader = {
     'Content-Type': 'application/json',
     'Accept': 'application/json'
   }
+  // if (sessionsStore.getTokenForCurrentSession) {
```

Comment:
you set the authorization header here based on the token that's in the store, but the auth header should probably be set based on which API is being called, right?

Reply:
yep